### ISSUE TITLE
[Issue]: rocWMMA 6.0.2 build with tests fails with "no member named 'cout' in namespace 'std'"

### ISSUE THREAD

**Issue by AngryLoki** [ROCm/rocWMMA#360]
### Problem Description

From build log:
```
FAILED: test/gemm/gemm_PGR1_LB2_MP0_MB_CP/test/wave/CMakeFiles/gemm_PGR1_LB2_MP0_MB_CP_WV-validate.dir/16x16_tt_1x1.cpp.o 
/usr/bin/cmake -E time /usr/bin/hipcc -DROCWMMA_VALIDATE_WITH_ROCBLAS -DROCWMMA_VALIDATION_TESTS -DUSE_PROF_API=1 -D__HIP_PLATFORM_AMD__=1 -I/var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm/gemm_PGR1_LB2_MP0_MB_CP/test/wave -I/var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test -I/var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm/gemm_PGR1_LB2_MP0_MB_CP -I/var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm -I/var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/library/include -isystem /usr/include/rocblas  -march=native -O2 -pipe -fdiagnostics-color=always -flto=thin -Werror=odr -Werror=strict-aliasing -std=c++17 -mcmodel=large -x hip --offload-arch=gfx908:xnack- -fopenmp=libomp -MD -MT test/gemm/gemm_PGR1_LB2_MP0_MB_CP/test/wave/CMakeFiles/gemm_PGR1_LB2_MP0_MB_CP_WV-validate.dir/16x16_tt_1x1.cpp.o -MF test/gemm/gemm_PGR1_LB2_MP0_MB_CP/test/wave/CMakeFiles/gemm_PGR1_LB2_MP0_MB_CP_WV-validate.dir/16x16_tt_1x1.cpp.o.d -o test/gemm/gemm_PGR1_LB2_MP0_MB_CP/test/wave/CMakeFiles/gemm_PGR1_LB2_MP0_MB_CP_WV-validate.dir/16x16_tt_1x1.cpp.o -c /var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm/gemm_PGR1_LB2_MP0_MB_CP/test/wave/16x16_tt_1x1.cpp
In file included from /var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm/gemm_PGR1_LB2_MP0_MB_CP/test/wave/16x16_tt_1x1.cpp:27:
In file included from /var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm/gemm_PGR1_LB2_MP0_MB_CP/test/test_includes.hpp:31:
In file included from /var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm/gemm_PGR1_LB2_MP0_MB_CP/detail/kernel_generator_impl.hpp:33:
In file included from /var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm/gemm_PGR1_LB2_MP0_MB_CP/detail/kernel_impl.hpp:30:
In file included from /var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm/gemm_PGR1_LB2_MP0_MB_CP/device/kernel_device_func.hpp:36:
In file included from /var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm/gemm_PGR1_LB2_MP0_MB_CP/device/kernel_predicates.hpp:30:
/var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm/gemm_predicates_base.hpp:92:18: error: no member named 'cout' in namespace 'std'
   92 |             std::cout << "Global Predicates:\n";
      |             ~~~~~^
/var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm/gemm_predicates_base.hpp:93:18: error: no member named 'cout' in namespace 'std'
   93 |             std::cout << "TBlockXTest: " << (bool)GlobalPredicates::TBlockXTest << std::endl;
      |             ~~~~~^
/var/tmp/portage/sci-libs/rocWMMA-6.0.2/work/rocWMMA-rocm-6.0.2/test/gemm/gemm_predicates_base.hpp:94:18: error: no member named 'cout' in namespace 'std'
   94 |             std::cout << "MinTBlockTest: " << (bool)GlobalPredicates::MinTBlockTest << std::endl;
      |             ~~~~~^
...
```

Please add  `#include <iostream>` to https://github.com/ROCm/rocWMMA/blob/develop/test/gemm/gemm_predicates_base.hpp (or remove prints).

### Operating System

Gentoo

### CPU

AMD Ryzen 7 8700G

### GPU

AMD Instinct MI300X

### ROCm Version

ROCm 6.0.0

### ROCm Component

rocWMMA

**Comment by CongMa13** [comment]
Hi @AngryLoki ! Just wanted to give you a heads up that the issue you reported has been fixed in version 6.1 and above. To ensure that you have the best possible experience, we suggest upgrading to the latest version. If everything works smoothly with the new version, we'd really appreciate it if you could close this ticket. As always, don't hesitate to get in touch if you have any further questions or concerns. Thanks!

**Comment by AngryLoki** [comment]
Builds fine now (checked with 6.1.1), thanks.
